posts/post-5.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <div>
+            <h1>Sellbrite Team Remix</h1>
+            <hr>
+
+            <style id="sb-team">
```

Comment:
this should be on the div haha

Reply:
made changes to this. Is it correct?

<div id="sb-team">
            <h1>Sellbrite Team Remix</h1>
            <hr>
        </div>
            <style>
                table, th, td {
                border: 1px solid black;
                border-collapse: collapse;
                }